src/Compiler/CompilerOptions.cpp

```diff
@@ -341,10 +367,26 @@ std::string getCompilerOption(const OptionKind kind) {
     return getTargetAccel();
   case OptionKind::CompilerOptLevel:
     return getOptimizationLevelOption();
-  case OptionKind::OPTFlag:
-    return getXoptOption();
-  case OptionKind::LLCFlag:
-    return getXllcOption();
+  case OptionKind::OPTFlag: {
+    std::vector<std::string> flags = getXoptOption();
+    std::string s;
+    for (int i = 0, n = flags.size(); i < n; ++i) {
+      s.append(flags.at(i));
+      if (i != n - 1)
+        s.append(",");
+    }
+    return s;
+  }
+  case OptionKind::LLCFlag: {
```

Comment:
There's a bit of code duplication here that I think could be reduced. Something like:

```
case LLC:
case OPT:
    std::vector flags = LLC ? getLLC : get OPT;
    ...
    
```